src/Client.ts

```diff
@@ -288,22 +308,31 @@ export default class Client {
 
     await Promise.all(
       topics.map(async (topic) => {
-        return this.publishEnvelope({
-          contentTopic: topic,
-          message: msgBytes,
-          timestamp,
-        })
+        return this.publishEnvelopes([
+          {
+            contentTopic: topic,
+            message: msgBytes,
+            timestamp,
+          },
+        ])
       })
     )
 
     return this.decodeMessage(msgBytes, topics[0])
   }
 
-  async publishEnvelope(env: {
-    contentTopic: string
-    message: Uint8Array
-    timestamp?: Date
-  }): Promise<void> {
+  async publishEnvelopes(envelopes: PublishParams[]): Promise<void> {
```

Comment:
Converted this to always accept an array of envelopes so we can send the invite to both the sender and recipient topics in one API request.